Source/WebCore/PAL/pal/PlatformMac.cmake

```
@@ -191,6 +192,7 @@ list(APPEND PAL_SOURCES
     cocoa/UsageTrackingSoftLink.mm
     cocoa/VisionKitCoreSoftLink.mm
     cocoa/WebPrivacySoftLink.mm
+    cocoa/WritingToolsUISoftLink.h
```

This should be the source file instead
```suggestion
    cocoa/WritingToolsUISoftLink.mm
```